src/ui/server/controllers/insurance/business/company-details/index.ts

```diff
@@ -0,0 +1,93 @@
+import { PAGES } from '../../../../content-strings';
+import { Request, Response, CompanyHouseResponse } from '../../../../../types';
+import { TEMPLATES, ROUTES, FIELD_IDS } from '../../../../constants';
+import corePageVariables from '../../../../helpers/page-variables/core/insurance';
+import api from '../../../../api';
+import companiesHouseValidation from './validation/companies-house';
+import companyHouseResponseValidation from './validation/companies-house-response';
+
+import { companyHouseSummaryList } from '../../../../helpers/summary-lists/company-house-summary-list';
+
+const { COMPANY_HOUSE } = FIELD_IDS.INSURANCE.EXPORTER_BUSINESS;
+
+const get = async (req: Request, res: Response) => {
+  return res.render(TEMPLATES.INSURANCE.EXPORTER_BUSINESS.COMPANY_DETAILS, {
+    ...corePageVariables({
+      PAGE_CONTENT_STRINGS: PAGES.INSURANCE.EXPORTER_BUSINESS.COMPANY_DETAILS,
+      BACK_LINK: req.headers.referer,
+    }),
+    POSTROUTE: ROUTES.INSURANCE.EXPORTER_BUSINESS.COMPANY_HOUSE_SEARCH,
+    FIELD_IDS: COMPANY_HOUSE,
```

Comment:
Can we use `FIELDS` object please like in other places - approach in place is:

- `FIELD` for a single field
- `FIELDS` object for multiple fields

This will also help with consuming "fields" content strings 

Also `POST_ROUTE` instead of `POSTROUTE` please

Lastly would be nice to have `PAGE_VARIABLES` object at the top of the controller as it saves duplication in get and post controllers.

Reply:
done